lib/bitex_bot/models/buy_opening_flow.rb

```diff
@@ -12,24 +12,27 @@ module BitexBot
   # @attr order_id The first thing a BuyOpeningFlow does is placing a Bid on Bitex, this is its unique id.
   #
   class BuyOpeningFlow < OpeningFlow
+    # BASE_COIN = Settings.bitex.orderbook.base
+    # QUOTE_COIN = Settings.bitex.orderbook.base
+    #
     # Start a workflow for buying bitcoin on bitex and selling on the other exchange. The amount to be spent on bitex is
-    # retrieved from Settings, if there is not enough USD on bitex or BTC on the other exchange then no order will be placed
-    # and an exception will be raised instead.
+    # retrieved from Settings, if there is not enough on QUOTE_COIN bitex or BASE_COIN on the other exchange then no order will
+    # be placed and an exception will be raised instead.
     #
     # The amount a BuyOpeningFlow will try to buy and the price it will try to buy at are derived from these parameters:
```

Comment:
The amount a BuyOpeningFlow will try to buy and the price it will *charge* to buy at are derived from these parameters ?